src/integrationTest/groovy/wooga/gradle/IntegrationSpec.groovy

```diff
@@ -45,10 +45,21 @@ class IntegrationSpec extends nebula.test.IntegrationSpec {
         new File(path).path
     }
 
+    // TODO: Move to gradle-commons
     Boolean fileExists(String... path) {
         fileExists(path.join("/"))
     }
```

Comment:
In this case, it calls the underlying nebula test method which does that check. If I am going to check a lot of files I would not want to write `new File(projectDir, #).exists()` repeatedly.

I will use System separator property, that was my miss.

Reply:
No you miss the point. The whole call with ("foo", "bar", "baz") to create a string "foo/bar/baz" is weird. What is the usecase and why would it be benefical to have it in the comons libary. Also `org.apache.commons.io.FileUtils` has the method `getFile` which could be used on the calling side to construct a `File` object from multiple strings.
What I want to get at is that we should not overload API and more specificly the commons libary with convinent overloads to construct objects from other forms. The library should use `File` objects whenever possible. And we can converters for calling said APIs. But a `fileExtist` or `createFile` etc method that takes all forms of parameter types that could be converted to a file object is just bloat. I know that nebular tests has them. But I never actually use them because I think they don't bring me much benefit over the vanilla API calls. 

```java
   /**
     * Construct a file from the set of name elements.
     *
     * @param names the name elements
     * @return the file
     * @since 2.1
     */
    public static File getFile(final String... names) {
        Objects.requireNonNull(names, "names");
        File file = null;
        for (final String name : names) {
            if (file == null) {
                file = new File(name);
            } else {
                file = new File(file, name);
            }
        }
        return file;
    }
```

Reply:
Yes